src/libmongoc/tests/mock_server/mock-server.c

```diff
@@ -1450,6 +1475,10 @@ mock_server_destroy (mock_server_t *server)
    int64_t deadline = bson_get_monotonic_time () + 10 * 1000 * 1000;
    request_t *request;
 
+   if (!server) {
+      return;
+   }
```

Comment:
Any reason for adding this? Did some test introduce a call to `mock_server_destroy` with a null argument, or was it just something you noticed and wanted to change for consistency?